actions.md

```diff
@@ -36,6 +36,12 @@ These are the core operations that users can request the system to perform.
   * **Required Parameters**: Cluster name, namespace name, workload kind (e.g., `Deployment`, `Pod`), workload name.
   * **Usage Example (CLI)**: `automated-actions openshift-workload-restart --cluster my-cluster --namespace my-namespace --kind Deployment --name my-app-deployment`
 
+* **`openshift-workload-delete`**:
+  * **Description**: Deletes a specified workload (e.g., ConfigMap, Job, Secret) in an OpenShift cluster.
+  * **Use Case**: Useful for cleaning up resources that are no longer needed, such as temporary jobs or outdated configurations.
```

Comment:
we should make it clear somewhere that if someone deletes a resource that is managed by a saas file, the resource will NOT be recreated auomtatically unless there is a new trigger (manual or automatic).

One the opposite, openshift-resources will resync after few minutes.